File: space-game/WebApi/Controllers/CameraController.vb
Imports System.Web.Http
Imports Camera

Namespace Controllers
    Public Class CameraController
        Inherits ApiController

        Private ReadOnly _cameraService As ICameraService

        Public Sub New(cameraService As ICameraService)
            _cameraService = cameraService
        End Sub

        <HttpGet()>
        Public Sub SetTarget(target As String)
            _cameraService.SetTarget(target)
        End Sub
    End Class
End Namespace

File: space-game/WebApi/Controllers/CameraController.cs
using Camera;
using System.Web.Http;

namespace WebApi.Controllers
{
    public class CameraController : ApiController
    {
        private readonly ICameraService _cameraService;
        public CameraController(ICameraService cameraService)
        {
            _cameraService = cameraService;
        }

        [HttpGet]
        public void SetTarget(string target)
        {
            _cameraService.SetTarget(target);
        }
    }
}
